lib/foreman_hooks/util.rb

```diff
@@ -8,14 +8,26 @@ def render_hook_type
       when Host::Managed
         'host'
       else
-        self.class.name.downcase
+        self.class.name.demodulize.downcase
```

Comment:
You changed the README to keep the namespace in model names, but this is still implemented?